### PR TITLE
Re-enables 'Pulling bleeding people causes more damage'

### PR DESCRIPTION
**Pull request by SonixApache**
wat r u, casull?
Re-enabled some of the things implemented in 67c5e8e3a80c7caf67f25ea3f74e1f978fd9b7c2

Discuss. There was some talk about adding splints to emergency lockers, or changing the oxygen kits found in o2 lockers into 'emergency medkits'. 

Tested, seemed to work fine.

:cl:
 - rscadd: The old 'pulling hurt people causes more damage' system has been re-enabled. this means:
 - tweak: Dragging someone who is hurt and knocked down will make their blood gush out and their wounds worsen.
 - tweak: Dragging someone who is in crit and bleeding / has broken limbs will cause massive damage and bloodloss.
- experimental: Roller beds are your best friend. Remember that you can fold them by dragging them to yourself.
- experimental: Stabilize your patient using splints and bandages/gauze/trauma kits before transport, or grab them and move them that way.

### CODE REVIEW

**Comment by kilozombie** [comment]
On a perfectly working station, this is great.
On a station where there are breaches everywhere and random shit is going on and you need every possible bit of time to get your friend to safety, this can feel... kinda crummy.
Not sure my thoughts on having this back.

**Comment by Lokiamis** [comment]
I'd like for there to be a way for borgs to transport injured people without harming them.

**Comment by kilozombie** [comment]
Does this mean you could hit a guy with a toolbox to break his skull, then drag him around to permanently damage the rest of his body?

**Comment by kol1th** [comment]
@kilozombie yes

**Comment by SonixApache** [comment]
In testing: Lesser wounding (when not in crit) generally occurs with such low frequency that the natural healing outheals it. The bloodloss is pretty strong, though. Not nearly as strong as normal bloodloss from bleeding limbs (or internal bleeding, god forbid), but still. It stacks.

@kilozombie Yup. Damage isn't that horrible, though. This just makes you die a lot faster when in crit.
@Lokiamis Rollerbeds will be your only option if you're not a medborg. Else just stabilize them first.

File: code/game/objects/effects/decals/Cleanable/humans.dm [hunk 1]
```diff
@@ -148,7 +148,7 @@ var/global/list/blood_list = list()
 
 
 
-
+//We should really get directional blood streak sprites again --snx
```
Comment (ghost) [inline code/game/objects/effects/decals/Cleanable/humans.dm:151]: We do have them.

blood.dmi tracks

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -703,6 +703,31 @@ Thanks.
...
+											H:vessel.remove_reagent("blood",2)
+								if(prob(6))
+									M.adjustBruteLoss(1)
+									M.visible_message("<span class='warning'>\The [M]'s wounds worsen from being dragged!</span>")
+							if(M.pull_damage())
+								if(prob(25))
+									M.adjustBruteLoss(2)
+									M.visible_message("<span class='danger'>\The [M]'s wounds worsen terribly from being dragged!</span>")
+									var/turf/location = M.loc
+									if (istype(location, /turf/simulated))
```
Comment (ghost) [inline code/modules/mob/living/living.dm]: Isn't there an isturf macro?

File: code/game/objects/effects/decals/Cleanable/humans.dm (hunk 1 above)
Comment (SonixApache) [reply code/game/objects/effects/decals/Cleanable/humans.dm:151]: Those are 'footprints' for wheelchairs. I'm talking about shit like gib1 and gib5, except much worse. I'll get you a picture from another codebase.

Comment (SonixApache) [reply code/game/objects/effects/decals/Cleanable/humans.dm:151]: Issue is that it doesn't have that transparent effect that we use for the multiple blood color types.

File: code/modules/mob/living/living.dm (hunk 2 above)
Comment (SonixApache) [reply code/modules/mob/living/living.dm]: done

**Comment by JMWTurner** [comment]
MAKE ROLLER BEDS GREAT AGAIN

**Comment by ghost** [comment]
Could we maybe have the ability to grab them and carry them at a cost of speed, so they don't face certain death if there isn't a medic on site?

File: code/game/objects/effects/decals/Cleanable/humans.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/effects/decals/Cleanable/humans.dm:151]: Just turn it to greyscale and implement coloring, not very hard.

**Comment by kilozombie** [comment]
...like, say, some sort of fourth "intent" with which you could "grab" people, perhaps with a UI icon tinted yellow, in the bottom right?
Yes, I like that idea.

**Comment by ghost** [comment]
Well I mean I'm assuming that still damages them since they'd still be on the ground, or at least it appears so.
Key word in that comment was *carry*.

**Comment by kilozombie** [comment]
>experimental: Stabilize your patient using splints and bandages/gauze/trauma kits before transport, or grab them and move them that way.

I... assume that this means grabbing is fair game, seeing as it's not dragging them along the ground, it's literally hoisting them up.

**Comment by ghost** [comment]
OH shit sorry, I didn't see that.
Disregard what I said, then.

:^(

**Comment by SonixApache** [comment]
Grabbing != pulling. Grabbing is fine.

Don't tell anyone, but lockers and office chairs work fine as well.

**Comment by AI-Is-Rogue** [comment]
Psst.


Body bags.

**Comment by invoid9** [comment]
Odysseus sleepers

ODYSSEUS SLEEPERS

**Comment by Kammerjunk** [comment]
Bring back hardcore bleeding woop woop
All who forget to order blood must suffer
All who die outside medbay must be so drained from pulling that you'd need an actual MD to treat them
I want this

**Comment by ghost** [comment]
Cloner will see more action than it already does.

Cloner nerfed when?

**Comment by Kurfursten** [comment]
@Skullyton When players who go into crit with a bleed automatically get bold red text that advises them to succumb immediately in order to conserve blood for medbay

**Comment by Blithering** [comment]
Also when putting a bleeding patient into cryo immediately extends a spike that decapitates the person doing so.

**Comment by SonixApache** [comment]
Medibots should let out a roar and inject cyanide into the culprit whenever they spot that happening

**Comment by jknpj** [comment]
I can port rollerbed holders for mediborgs if borgs 'helping' become a issue.

**Comment by Probe1** [comment]
Polled players in game, support was pretty close to unanimous.

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -703,6 +703,29 @@ Thanks.
 							pulling.Move(T, get_dir(pulling, T))
 							if(M && secondarypull)
 								M.start_pulling(secondarypull)
+						//this is the gay blood on floor shit -- Added back -- Skie
+							if (M.lying && (prob(M.getBruteLoss() / 6)))
```
Comment (Probe1) [inline code/modules/mob/living/living.dm]: The probability needs increased drastically.  It is extremely difficult to get to proc.

Comment (SonixApache) [reply code/modules/mob/living/living.dm]: Got suggestions? / 2 ?

**Comment by Probe1** [comment]
Its still piss weak but lets start from here and see how it goes in game.

**Comment by SonixApache** [comment]
aww yeah

**Comment by SonixApache** [comment]
oh goddamnit, the last two lines of the changelog didn't parse because it's "experiment" and not "experimental"

**Comment by Kammerjunk** [comment]
i told u i was hardcore...

**Comment by Probe1** [comment]
just make a PR with a cl yml file @SonixApache

**Comment by SonixApache** [comment]
TOO LATE, WE'RE A DAY PAST. IT'S OVER.

**Comment by coldcola** [comment]
fucking.
blood.
everywhere.

**Comment by JMWTurner** [comment]
Just make it stop painting blood once the body has less than a certain amount of blood in it.

**Comment by Exxion** [comment]
Oh dear god I didn't see that part of the PR
Time for lag out the ass
(For reasons unknown, blood in large quantities causes massive amounts of clientside lag. In my experience,  more than anything else in the game.)

**Pull request merged**